src/app/core/models/templates/sections/tester-response-section-parser.model.ts

```diff
@@ -0,0 +1,118 @@
+const { char, choice, coroutine, everyCharUntil, letters, lookAhead, optionalWhitespace, possibly, str, whitespace } = require('arcsecond');
+
+const SECTION_TITLE_PREFIX = '## :question: Issue ';
+const TEAM_CHOSE_PREFIX = 'Team chose ';
+const TESTER_CHOSE_PREFIX = 'Originally ';
+const DISAGREEMENT_REASON_PREFIX = '**Reason for disagreement:** ';
+const LINE_SEPARATOR = '-------------------';
+const DUPLICATE_STATUS_MESSAGE =
+  "Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)";
+
+function buildExtractResponseParser(category: string) {
+  return coroutine(function* () {
+    yield str('[`' + category + '.');
+    const response = yield letters;
+    yield str('`]');
+    return response;
+  });
+}
+
+function buildTeamResponseParser(category: string) {
+  const extractResponseParser = buildExtractResponseParser(category);
+
+  return coroutine(function* () {
+    yield str(TEAM_CHOSE_PREFIX);
+    const teamChose = yield extractResponseParser;
+    return teamChose;
+  });
+}
+
+function buildTesterResponseParser(category: string) {
+  const extractResponseParser = buildExtractResponseParser(category);
+
+  return coroutine(function* () {
+    yield str(TESTER_CHOSE_PREFIX);
+    const testerChose = yield extractResponseParser;
+    return testerChose;
+  });
+}
+
+export const DisagreeCheckboxParser = coroutine(function* () {
+  yield str('- [');
+  const disagreeCheckboxChar = yield choice([char('x'), whitespace]);
+  yield str('] I disagree');
+
+  return disagreeCheckboxChar === 'x';
```

Comment:
See if 
``` between((str('- ['))(str('] I disagree')(choice()).map(checkbox => checkbox === 'x')```
looks better.

Reply:
<img width="804" alt="Screenshot 2022-03-21 at 2 25 23 PM" src="https://user-images.githubusercontent.com/68138671/159213872-8511761e-cd0b-4854-ac65-0bcae05c2d4b.png">

don't think it looks better. also for some reason i can't get it to work - arcsecond doesn't recognise it as a parser?

Reply:
I think its because the parser is `char('x')` not `char`.
In fact I think it is possible to further abstract a CheckboxParser

```suggestion
pipeParsers([
  checkboxParser,
  skip(str(' I disagree'))
])
const checkboxParser = between(str('- ['))(char(']')(choice()).map(checkbox => checkbox === 'x')
```

so the DoneParser can just be 
```pipeParsers([
  checkboxParser,
  skip(str(' Done'))
])```

For the checkboxParser itself I'll leave it to your discretion, the coroutine form is fine with me.